blueprints/blog-post.yaml

```
@@ -0,0 +1,25 @@
+title: Blog Post
+'@extends': default
+
+form:
+  fields:
+    tabs:
+      fields:
+        buttons:
+          type: tab
+          title: Blog Post
+          fields:
+          
+            header.image:
+                  type: filepicker
+                  label: Summary Image
+                  folder: 'self@'
+                  preview_images: true
+                  description: Choose from an image uploaded to the Page Media section of the Content tab.

```

Once this gets moved into the Content tab we can go ahead and change the description.
```suggestion
                  description: Choose from an image uploaded to the Page Media section..
```